src/text_buffer.rs

```diff
@@ -4,16 +4,31 @@
 
 use libc::{c_char, c_int};
 use std::boxed::Box as Box_;
-use std::mem::transmute;
+use std::mem::{self, transmute};
 use std::{slice, str};
 use ffi;
+use gdk;
 use glib::translate::*;
 use glib::signal::connect;
 use glib_ffi;
 use TextBuffer;
 use TextIter;
 
 impl TextBuffer {
+    pub fn get_deserialize_formats(&self) -> Vec<gdk::Atom> {
+        unsafe {
+            let mut n_formats = mem::uninitialized();
+            FromGlibPtrContainer::from_glib_container(ffi::gtk_text_buffer_get_deserialize_formats(self.to_glib_none().0, &mut n_formats))
```

Comment:
Then what should the `num` argument be when calling it? The total number is returned by the type not specified by the caller, which is why I left it as is. But the docs of `from_glib_container_num` are sparse.